### ISSUE TITLE
Style Reset

### ISSUE THREAD

**Issue by HipsterBrown** [namely/styleguide#281]
Some styles have been affected by the [`_reset.scss`](https://github.com/namely/styleguide/blob/develop/src/scss/base/_reset.scss) when adding the `.namely-ui`class to surrounding parent elements on HCM pages. 

Example Pages in HCM:
https://namely.namely-pr-4388.broadway.namely.tech/performance/goals/admin/language
https://namely.namely-pr-4388.broadway.namely.tech/performance/goals/admin/settings

This is an open discussion for how to resolve this issue.

**Comment by jasonfellows** [comment]
Yeah, if we keep using this, then we have to reach higher specificity than class > element for ALL nodes within the .namely-ui namespace going forward. This is troublesome for a lot of reasons.

The specificity of `.namely-ui element` is higher than `.classname`, so we're seeing a lot of places on HCM where layouts collapse completely when they're put inside the namespace.

The more we use .namely-ui with this reset in place, the greater the effort it takes to extract it. I think we should remove this with the theme PR: https://github.com/namely/styleguide/pull/274/files

Here is another example of what is happening in the dark corners of the site we never clean... 
<img width="302" alt="screen shot 2016-04-12 at 5 47 45 pm" src="https://cloud.githubusercontent.com/assets/1119770/14526881/9daf0d34-0246-11e6-994c-c3b2701096a6.png">
